lib/modules/themes.nix

```diff
@@ -102,7 +102,7 @@ rec {
     options = {
       # TODO: Get this directly from home-manager instead of copying it
       plugins = mkOption {
-        type = listOf (either package neovim.pluginWithConfigModule);
+        type = neovim.pluginWithConfigModule;
```

Comment:
Why are we not allowing a package anymore?

Reply:
The type itself that we're now getting from home-manager is a list of either packages or plugin with config module. See it 👉🏽 https://github.com/nix-community/home-manager/blob/94780dd888881bf35165dfdd334a57ef6b14ead8/modules/programs/neovim.nix#L281

Reply:
Ah yes indeed, great!